configs/containers/docker-ubuntu-gcc-openmpi.yaml

```diff
@@ -114,5 +97,5 @@ spack:
 
     # Labels for the image
     labels:
-      app: "@SPEC@"
+      app: ""
```

Comment:
Not sure if this label is needed at all (that came from the spack template for containers), but if it is, is there a way to set it to the spec?

Reply:
Yes, I meant to add this. Just a key insert.

Reply:
Fixed